scipy/linalg/tests/test_decomp.py

```diff
@@ -1217,6 +1217,7 @@ class TestSVD_GESVD(TestSVD_GESDD):
 # Allocating an array of such a size leads to _ArrayMemoryError(s)
 # since the maximum memory that can be in 32-bit (WASM) is 4GB
 @pytest.mark.skipif(IS_WASM, reason="out of memory in WASM")
+@pytest.mark.xfail_on_32bit("out of memory in 32-bit CI workflow")
```

Comment:
@ev-br thought I should ping you here. Is it OK to skip this on 32-bit? Even after the [other work](https://github.com/scipy/scipy/blame/1dd806b71cb952e563dfa187a3ec77e745501b59/scipy/linalg/tests/test_decomp.py#L1220) adjusting this test, it's [still failing](https://github.com/scipy/scipy/actions/runs/18619998392/job/53097937434) on the 32-bit workflow.

Reply:
Yes, fine to skip this test wherever it OOMs.
Either like you do here, or with `check_free_memory`, similar to https://github.com/scipy/scipy/blob/1dd806b71cb952e563dfa187a3ec77e745501b59/scipy/linalg/tests/test_basic.py#L2133
, whichever works. Thanks for taking care of these CI failures!